src/fuddly/framework/plumbing.py

```diff
@@ -759,125 +758,159 @@ def monitor_probes(self, prefix=None, force_record=False):
 
     @EnforceOrder(initial_func=True, final_state="get_projs")
     def get_data_models(self, fmkDB_update=True):
+        self._get_data_models_from_fs(fmkDB_update)
+        self._get_data_models_from_modules(fmkDB_update)
+
+        if fmkDB_update:
+            self.fmkDB.insert_data_model(Database.DEFAULT_DM_NAME)
+            self.fmkDB.insert_dmaker(Database.DEFAULT_DM_NAME, Database.DEFAULT_GTYPE_NAME,
+                                     Database.DEFAULT_GEN_NAME, True, True)
+
+    def _get_data_models_from_fs(self, fmkDB_update=True):
+        if not self._quiet:
+            self.print(colorize(FontStyle.BOLD + "=" * 63 + "[ Data Models (filesystem) ]==", rgb=Color.FMKINFOGROUP))
+
         data_models = collections.OrderedDict()
 
         def populate_data_models(path, prefix=""):
```

Comment:
why defining a function inside another to use it only once
--> get out or remove function